src/config.rs

```diff
@@ -121,6 +121,8 @@ impl PyModel {
 #[pyproto]
 impl PyObjectProtocol for PyModel {
     fn __repr__(&self) -> PyResult<String> {
-        Ok(format!("{:?}", self.config.borrow()))
+        Ok(format!("{:?}", self.config.borrow().model))
+    }
```

Comment:
superfluous closing character

Reply:
Oops, cherry picked this from another set of changes. Thanks!

Reply:
Fixed!